yar-guice/src/main/java/org/javabits/yar/guice/BlockingSupplierImpl.java

```diff
@@ -12,7 +12,7 @@
 import static com.google.common.util.concurrent.Futures.getUnchecked;
 import static java.util.concurrent.CompletableFuture.completedFuture;
 
-class BlockingSupplierImpl<T> implements BlockingSupplier<T>, SupplierListener, SupplierWrapper<T> {
+class BlockingSupplierImpl<T> implements BlockingSupplier<T>, SupplierListener, SupplierWrapper<T>, com.google.common.base.Supplier<T> {
```

Comment:
We should probably discuss this... I don't understand why it work with java8 and not with java11